src/aux-runtime/runtime/AuxLibrary.spec.ts

```diff
@@ -4523,6 +4524,12 @@ describe('AuxLibrary', () => {
                 expect(result).toEqual('inst');
             });
 
+            it('should return array when multiple inst are loaded', () => {
+                player.tags.inst = array();
+                const result = library.api.os.getCurrentInst();
+                expect(result).toEqual(array);
```

Comment:
@TroyceGowdy This should test that `os.getCurrentInst()` returns the first element of the array, not that it returns the array. Also calling a function named `array()` doesn't actually give you an array because the `array` function is imported from `@hapi/joi` above, which handles schema validation - not array creation. Prefer using [JavaScript Arrays](https://javascript.info/array) instead of imported objects & functions.